sfft_and_animate.py

```diff
@@ -91,15 +91,21 @@ def sfft_and_animate(oid,band):
     for i, path in enumerate(filepaths):
         skysub = sky_subtract(path=path)
         skysub_ref_init.append(skysub)
-        align = imalign(template_path=skysub_ref_init[0],sci_path=skysub)
+        align = imalign(template_path=skysub_ref_init[0],sci_path=skysub,force=True)
         imalign_ref_init.append(align)
 
         # Check that each reference tile contains an overlap_size cutout over the SN location.
         # This prevents PSF discontinuities too close to the SN. 
         with fits.open(align) as hdu:
+            print('ALIGN_PATH')
```

Comment:
Someday wrap these in a `debug` or `verbose` for the logging.

Reply:
Yes, definitely, for now they are intended as debugging statements but wrapping them into verbose is a good idea.